.github/CODEOWNERS

```
@@ -21,31 +21,32 @@
 # NOTE: Must be placed last to ensure enforcement over all other rules
 
 # Protection Rules for Github Configuration Files and Actions Workflows
-/.github/                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts-managers
+/.github/                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts-managers @hashgraph/hedera-smart-contracts-product
+/.github/workflows/.                            @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts-managers @hashgraph/hedera-smart-contracts-product @hashgraph/tuum-tech-hedera 
```

Other workflows use the structure `/.github/workflows/` without a "." at the end of the line. I'm not sure if this will break anything, but I suggest following the standard that we have in other files.

```suggestion
/.github/workflows/                            @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts-managers @hashgraph/hedera-smart-contracts-product @hashgraph/tuum-tech-hedera 
```